src/model/transformations/collected-variable.js

```diff
@@ -265,6 +266,28 @@ function getCollectedScope(questionsLoop, id, componentsStore) {
   });
   return isfound;
 }
+
+function getTableDynamique(componentsStore, id) {
+  let tableId = '';
+  Object.values(componentsStore)
+    .filter(
+      components =>
+        components.type === QUESTION &&
+        components.responseFormat.type === TABLE &&
+        components.responseFormat.TABLE.PRIMARY.type === LIST,
+    )
+    .map(component => {
+      if (
+        component &&
+        component.collectedVariables &&
+        component.collectedVariables.includes(id)
```

Comment:
You can remove linke 281 and 282, and do this changes

```suggestion
        component?.collectedVariables?.includes(id)
```